assets/src/components/button/style.scss

```diff
@@ -0,0 +1,53 @@
+/**
+ * Button styles,
+ */
+
+.muriel-button {
+	text-align: center;
+	font-size: 14px;
+	line-height: 18px;
+	font-weight: 500;
+	color: $muriel-gray-700;
+	text-shadow: none;
+
+	&.is-button {
+		display: inline-block;
+		background-color: $muriel-white;
+		border-color: #CCCCCC;
+		border-radius: 3px;
+		cursor: pointer;
+		height: auto;
+		padding: 13px;
```

Comment:
The buttons outside of the card seem a little tight, maybe could use more horizontal padding? I bring this up mostly because I'm working on the Checklist component right now which requires non-full-width buttons, and in the mockups these have a bit more horizontal spacing. First screenshot is what I see on this branch and second is from Figma:

<img width="295" alt="Screen Shot 2019-05-14 at 8 22 37 AM" src="https://user-images.githubusercontent.com/1477002/57697635-bab60480-7621-11e9-9240-7beec47f1461.png">

<img width="1055" alt="Screen Shot 2019-05-14 at 8 25 02 AM" src="https://user-images.githubusercontent.com/1477002/57697689-cefa0180-7621-11e9-9caa-82a5d76bf06b.png">

Reply:
Yep, that makes sense. In [88826d5c343b4e580c6b7473704697683db34532] I've given more horizontal padding to those buttons, and reduced the vertical padding 'cause they looked weirdly fat otherwise. The centered buttons are then given different padding.